plasmapy/particles/particle_class.py

```diff
@@ -678,20 +683,42 @@ def json_dict(self) -> dict:
             '__init__': {'args': ('e-',), 'kwargs': {}}}}
         """
         particle_dictionary = super().json_dict
-        particle_dictionary["plasmapy_particle"]["__init__"]["args"] = (self.particle,)
+        particle_dictionary["plasmapy_particle"]["__init__"]["args"] = (self.symbol,)
         return particle_dictionary
 
     @property
     def particle(self) -> Optional[str]:
         """
         Return the particle's symbol.
 
+        .. deprecated:: 0.6.0
+            `Particle.symbol` has been deprecated and will be removed in
+            a subsequent release of PlasmaPy.  Use `Particle.particle`
+            instead.
+
         Examples
         --------
         >>> electron = Particle('electron')
         >>> electron.particle
         'e-'
         """
+        warnings.warn(
+            "Particle.particle has been deprecated and will be removed in "
+            "a subsequent release. Use Particle.symbol instead."
+        )
+        return self.symbol
+
+    @property
+    def symbol(self) -> Optional[str]:
+        """
+        Return the particle's symbol.
```

Comment:
Hm... "molecular" typically refers to a particle containing two or more atoms, so it's not quite the word I'm looking for with the current version of `Particle`.  With that said, it is a good point that this could use a bit of clarification, so I'll think about how to reword it.  Maybe `Return the symbol of the particle, atom, isotope, or ion.`

Reply:
> Maybe `Return the symbol of the particle, atom, isotope, or ion.`

I like that suggestion over the original.  Another possible idea, `Return a symbolic representation of the particle (e.g. the atomic symbol).`